src/gui/advancedsettings.cpp

```diff
@@ -810,6 +812,10 @@ void AdvancedSettings::loadAdvancedSettings()
     // Tracker port forwarding
     m_checkBoxTrackerPortForwarding.setChecked(pref->isTrackerPortForwardingEnabled());
     addRow(TRACKER_PORT_FORWARDING, tr("Enable port forwarding for embedded tracker"), &m_checkBoxTrackerPortForwarding);
+    // Python executable path
+    m_pythonExecutablePath.setPlaceholderText(tr("(Auto detect if empty)"));
+    m_pythonExecutablePath.setText(pref->getPythonExecutablePath().toString());
+    addRow(PYTHON_EXECUTABLE_PATH, tr("Python executable path (requires restart)"), &m_pythonExecutablePath);
```

Comment:
The `requires restart` isn't always. There are some cases that simply reloading the search tab will have immediate effect. But in order to avoid usage confusion, I decided to add this wording.
And pardon, I currently can't afford the time to redesign and make the experience better. Hopefully the current form will be enough for those who demands it.